evadb/executor/ray_utils.py

```diff
@@ -26,8 +22,10 @@ class StageCompleteSignal:
     pass
 
 
-@ray.remote(num_cpus=0)
-def ray_wait_and_alert(tasks: List[ray.ObjectRef], queue: Queue):
+def ray_wait_and_alert(tasks, queue):
```

Comment:
Did you try running Ray enabled? I think this won't work if you remove the decorator. 

Is this a good solution? https://stackoverflow.com/questions/16192448/conditional-definition-of-a-function-in-a-module

Reply:
https://docs.ray.io/en/latest/ray-core/api/doc/ray.remote.html

This seems like a simpler solution: `f_with_2_gpus = f.options(num_gpus=2) `

Reply:
I think if we specify `num_gpus` on the Ray side, this (https://github.com/georgia-tech-db/eva/pull/817) issue may occur.